static/js/whyis_vue/components/pages/vega/editor/vega-editor.vue

```diff
@@ -1,3 +1,3 @@
 <template src="./vega-editor.html"></template>
 <script src="./vega-editor-script.js"></script>
-<style scoped lang="scss" src="./vega-editor.scss"></style>
+<style scoped lang="scss" src="../../../../assets/css/main.scss"></style>
```

Comment:
This seems a bit odd.  Adding the main stylesheet as a scoped style for a component.
I would recommend having a scoped stylesheet for each component as I did in vega-editor.scss.
see https://vue-loader.vuejs.org/guide/scoped-css.html